docs/getting-started/cluster-setup.md

```diff
@@ -0,0 +1,232 @@
+# Setting Up a Cluster
+You can setup a cluster in [Kubernetes](http://kubernetes.io/docs/whatisk8s/) or Jenkins.
+
+## Kubernetes
+
+### Installing kubectl
+To deploy and manage applications on Kubernetes, you’ll use the Kubernetes command-line tool, __kubectl__. It lets you inspect your cluster resources, create, delete, and update components, and much more.
+
+#### Prerequisites
+- [brew](http://brew.sh/)
+
+#### Install kubectl
+Run:
+
+```bash
+$ brew install kubernetes-cli
+```
+
+### Create your cluster
+A cluster consists of a master API server and a set of worker VMs called nodes.
+
+#### Prerequisites
+- an [AWS](http://aws.amazon.com) account
+- [AWS CLI](https://aws.amazon.com/cli/)
+
+#### Using get-kube
+This process takes about 5 to 10 minutes. User credentials and security tokens are written in `~/.kube/config`.
+By default, the script will provision a new VPC and a 4 node k8s cluster in `us-west-2a` (Oregon) with EC2 instances running on Debian.
+
+NOTE: You can override the variables defined in `config-default.sh` to change this behavior. See the [Getting Started with Kubernetes guide](http://kubernetes.io/docs/getting-started-guides/aws/) for more details.
+```bash
+#Using wget
+export KUBERNETES_PROVIDER=aws; wget -q -O - https://get.k8s.io | bash
+#Using cURL
+export KUBERNETES_PROVIDER=aws; curl -sS https://get.k8s.io | bash
+```
+
+
+### Create a Service
+A Kubernetes [Service](http://kubernetes.io/docs/user-guide/services/) is an abstraction which defines a logical set of Pods and a policy by which to access them - sometimes called a micro-service. The set of Pods targeted by a Service is (usually) determined by a Label Selector.
+
+Create a `service.yaml` file:
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: sdapi
+  labels:
+    app: screwdriver
+    tier: api
+spec:
+  type: LoadBalancer
+  ports:
+  - port: 80
+    targetPort: 8080
+  selector:
+    app: screwdriver
+    tier: api
+```
+
+Run the kubectl create command:
+```bash
+$ kubectl create -f service.yaml
+```
+
+
+### Setup secrets
+First you can create your own `default.yaml` file and fill out the constants with your own secrets.
```

Comment:
Also, I feel like this needs 2 things:

- A more thorough paragraph explaining which secrets are needed and why
- How to get ahold of the secrets should show up before the yaml file that uses them